CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.4.10 (TBD)
+
+- Bugfix: Fixed a bug where --json flag would not trigger for "telepresence list" for an empty list.
```

Comment:
nit: formatting `telepresence list` 

I'd also change the messaging to say something like "Fixed a bug where `--json` flag did not output json for `telepresence list` when there were no workloads" or something like that.